Report/sections/2_design/Design_evaluation.tex

```diff
@@ -0,0 +1,46 @@
+\section{Design evaluation} \label{chap:design evaluation}
+What is significant about developing a language is to carefully consider various parameters that could be around the language itself and the setting up of essential criteria for the programming language. 
+Therefore, it is crucial to be able to evaluate the different parameters using different tools. Thus, the group has chosen to use an already set up table, which covers all the most important criteria for setting up a programming language. 
```

Comment:
tror ikke helt det er underforstået hvad 'table' er.

Måske skrive noget ala 'use an already existing set of critera...'